src/Scene/NodeProcess.js

```diff
@@ -479,16 +479,19 @@ NodeProcess.prototype.processNode = function processNode(node, camera, params) {
  * @returns {NodeProcess_L7.NodeProcess.prototype.frustumCullingOBB.node@pro;camera@call;getFrustum@call;intersectsBox}
  */
 
-var quaternion = new THREE.Quaternion();
+const frustum = new THREE.Frustum();
+const obbMatrixWorld = new THREE.Matrix4();
 
 NodeProcess.prototype.frustumCullingOBB = function frustumCullingOBB(node, camera) {
-    // position in local space
-    var position = node.OBB().worldToLocal(camera.position().clone());
-    position.z -= node.distance;
+    // Move camera in OBB local space
+    obbMatrixWorld.copy(camera.camera3D.matrixWorldInverse);
```

Comment:
Now there's :
`this.scene3D.autoUpdate = false;`

I don't know which is priority between `scene.autoUpdate` and `object.matrixAutoUpdate`

I have to do update matrix world since this change, that's why I want to know where is the update matrix world (for not RTC) in the 239